src/Media/sass/main.css

```diff
@@ -26,139 +26,155 @@
   border: none;
   background: none;
 }
-
-.rightMain {
-  background-color: #fff;
+.rightSideBox .rightMain {
+  background-color: #ffffff;
   padding: 10px;
   margin: 20px;
   position: relative;
 }
-.rightMain .info {
+.rightSideBox .rightMain .info {
   font-weight: 400;
   margin: 10px;
 }
-.rightMain .heading {
+.rightSideBox .rightMain .heading {
   font-weight: 300;
   margin: 10px;
 }
-.rightMain .outerContainer {
+.rightSideBox .rightMain .outerContainer {
   position: relative;
+  width: inherit;
 }
-.rightMain .leftContainer {
+.rightSideBox .rightMain .outerContainer .leftContainer {
   width: 50%;
   height: inherit;
-  border: 1px solid #EAEBED;
+  border: 1px solid #eaebed;
   margin: 10px;
   display: inline-block;
 }
-.rightMain .rightContainer {
+.rightSideBox .rightMain .outerContainer .leftContainer .sliders {
+  margin: 40px 0px 40px 30px;
+  width: 90%;
+  width: 90%;
+}
+.rightSideBox .rightMain .outerContainer .leftContainer .grid {
+  position: relative;
+}
+.rightSideBox .rightMain .outerContainer .leftContainer .gridItem {
+  position: absolute;
+  right: 0;
+}
+.rightSideBox .rightMain .outerContainer .leftContainer .inputBox {
+  background-color: #f1f5fa;
+  width: 90%;
+}
+.rightSideBox .rightMain .outerContainer .leftContainer .errField {
+  color: #df2e38;
+  padding: 3px;
+  font-size: 14px;
+  margin: 2px 0px;
+  text-align: right;
+}
+.rightSideBox .rightMain .outerContainer .rightContainer {
   position: absolute;
```

Comment:
why added inline nesting

Reply:
Ma'am we have written the css in main.scss with nesting as you have explained and Live Sass Compiler converted it into main.css like this .